.github/CODESTYLE.md

```diff
@@ -0,0 +1,305 @@
+# WRF-Hydro Fortran Conventions Guide
+
+# Fortran standard
+New code contributions should be written to at least Fortran 2003 standard.
+
+# Source Code Formatting
+* Max line length 100 characters. All wrapped code statements shall at least be indented one additional level beyond the previous level.
```

Comment:
There's an irony in the fact that this line is well over 100 chars long 😛

Reply:
Code wrapping in editor was not set to hard wrap. This has been changed